packages/@aws-cdk/aws-ec2/test/integ.everything.expected.json

```diff
@@ -119,7 +134,8 @@
         },
         "SubnetId": {
           "Ref": "VPCPublicSubnet2Subnet74179F39"
-        }
+        },
+        "Tags": []
```

Comment:
Do we care about the empty Tags here? I can remove this I think by simply returning `undefined` if the list is empty, but is that the right path?